app/kubernetes/components/datatables/application/containers-datatable/containersDatatable.html

```diff
@@ -140,11 +163,13 @@
                   ui-sref="kubernetes.applications.application.stats({ pod: item.PodName, container: item.Name })"
                   style="margin-right: 10px"
                 >
-                  <i class="fa fa-chart-area" aria-hidden="true"></i> Stats
+                  <pr-icon icon="'pie-chart'" class="mr-1" feather="true"></pr-icon>Stats
```

Comment:
I noticed there's quite a few instances where `aria-hidden="true"` is being dropped when you swapped from `<i>` to `<pr-icon>`. The new UI component guide seems to say we should keep them. I believe it makes it so screen readers will ignore the icon which seems good. Is there any reason you've been removing them? Is it already included in pr-icon or something?

Reply:
being added by pr-icon component